peachjam/models/judgment.py

```diff
@@ -72,10 +72,31 @@ def __str__(self):
         return self.name
 
 
+class CourtRegistry(models.Model):
+    court = models.ForeignKey(
+        Court,
+        on_delete=models.CASCADE,
+        null=True,
+        related_name="court_registry",
+    )
+    name = models.CharField(_("name"), max_length=1024, null=False, blank=False)
+
+    class Meta:
+        verbose_name = _("court registry")
+        verbose_name_plural = _("court registries")
+        unique_together = ("court", "name")
+
+    def __str__(self):
+        return f"{self.name} Registry - {self.court}"
+
+
 class Judgment(CoreDocument):
```

Comment:
When saving a judgment, if there is a registry set, then also force the court to be `self.registry.court`, so that they always align. Do it at the start of `save()`

Reply:
Note to self: So that we don't have a situation whereby a judgment is assigned a registry that doesn't belong to its court.